content/en/docs/contribute/style/style-guide.md

```diff
@@ -358,6 +359,69 @@ Shortcodes inside include statements will break the build. You must insert them
+  <li> Use horizontal rules for decoration.</li></ul>
+</td></tr>
+</table>
+
+### Links
+ <table>
+  <tr><th>Do</th><th>Don't</th></tr>
+  <tr><td><ul><li>Links should provide context for the content they link to. Certain ports are open on your machines. See <a href="#check-required-ports">Check required ports</a> for more details.</li> <li>Use Markdown-style links ([link text](URL)).</li></ul></td><td><ul><li>Use ambiguous terms like “click here”. Certain ports are open on your machines. See <a href="#check-required-ports">here</a> for more details.</li> <li>Use HTML style links (&#60;link href="/media/examples/link-element-example.css" target="_blank"&#62;Visit our tutorial!)</li><li>Create links that open in new tabs or windows. For example, [example website](https://example.com){target="_blank"}</li></ul>
+</td></tr>
+</table>
+
+### Lists
+Grouping related items not just looks good visually, but also helps the non-sighted users. When a screen reader comes across a list—whether it is an ordered or unordered list—it will be announced to the user that there is a group of list items. The user can then use the arrow keys to cursor up and down between the various items in the list.
+Website navigation links can also be marked up as list items; after all they are nothing but a group of related links.
+
+ - Put a period at the end of every list item. 
+ - Uses dashes (-) for unordered lists. 
+ - Use the number one (1.) for ordered lists. 
+ - Use one space after a list item. 
+ - Indent nested lists with four spaces (for example, ⋅⋅⋅⋅-). 
+ - List items may consist of multiple paragraphs. Each subsequent paragraph in a list item must be indented by either four spaces or one tab.
+
+### Code
+
+<table>
+  <tr><th>Do</th><th>Don't</th></tr>
+  <tr><td><ul><li>Use single backticks to enclose inline code. For example, `var example = true`.</li><li>Use triple backticks before and after a multi-line block of code for fenced code blocks.</li><li> Use meaningful variable names that have a context.</li><li>Ensure that there are no trailing spaces in the code, unless it is absolutely neccesary.</li></ul></td><td><ul><li>Use multi-line blocks of code to create diagrams, flowcharts, or other illustrations.</li><li>Use variable names such as 'foo','bar', and 'baz' that are not meaningful and lack context.</li>
+  </ul>
+</td></tr>
+</table>
+
+{{< note >}} Syntax highlighting in the code block should conform to the [contrast guidelines.](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=141%2C143#contrast-minimum)
+
+### Tables
+
+The semantic purpose of a data table is to present tabular data. Sighted users can quickly scan the table but a screen reader goes through line by line. Proper markup must be added to help the screen reader make the correct associations that a sighted user would.
```

Comment:
@Rajakavitha1: With this PR, it seems like the style guide mixes general instructions for authors with accessibility specific instructions. You also don't mention the concept of `accessibility` on the page, unless I missed it. Would it be possible to add a section in the style guide just for accessibility? You may want to provide a link to the accessiblity standards or provide rationale for how the k8s docs are incorporating accessibility. Are these guidelines or requirements?
- In a list of items, is consistency with end punctuation more important than using `a period at the end of each list item`?
- A couple of the table columns/rows are not aligned. See the Links table for an example. 
- nit: `Grouping related items not just looks good visually, but also helps the non-sighted users`. Is `non-sighted` the right term or `visually-impaired users`? I don't know, just wondering.

Reply:
Thanks @kbhawkey !!!! You are right!!! The idea was to make the current guidelines that we have for writing no different than the accessibility guidelines to write content. We did not want to convey that the accessibility guidelines are good to have or optional. We want to treat these just like the other guidelines that we have.  It might be worth mentioning about writing guidelines to ensure accessibility. I tried to provide the rationale for the guidelines in Line breaks, Headings, Lists, and table. These sections provide guidelines for writing good content in general and address accessibility in specific. The rest of the sections anyways apply in general and address accessibility as well.

In the description, I did provide the link to the accessibility standards that were adopted to Markdown elements (the accessibility level A and AA guidelines only):
https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&currentsidebar=%23col_overview&levels=aaa

-  Yes, you are right in the list items, consistency with end punctuation is important. Not just the period. Could you please help me adopt the guidelines here to our markdown elements (http://web.mit.edu/course/21/21.guide/period.htm). 

- Yes, you are right the tables have just one row with the list of Do and Don't. I did not use separate row for each of the guideline because some of these Do guidelines did not have the corresponding Don't for example: "Ensure that there are no trailing spaces in the code, unless it is absolutely necessary". (Because the screen reader would even read out that space within the code.) Please suggest a better format to present Do and Don't.

- Yes, you are right it is best to avoid the word non-sighted or any other terms that refers to the sight disability. Reworded as suggested.

Reply:
A few more suggestions about front matter and new tables:

Hugo uses [front matter](https://gohugo.io/content-management/front-matter/#readout) to create metadata about a given documentation page. The Kubernetes documentation includes attributes such as [author, title, categories?, weight, and content_template].

```yaml
---
# basic front matter for content, task, tutorial
title:
content_template:
weight:
---
```

```yaml
---
# sample front matter for tutorial
reviewers:
- reviewer1
- reviewer2
title: Runtime Class
content_template: templates/tutorial
weight: 40
---
```
- May want to link to the other style/write-new-topic page containing info about [front matter](https://kubernetes.io/docs/contribute/style/write-new-topic/#adding-the-topic-title-to-the-front-matter), or front matter content to that page? 
- Could we link to a sample from each type of page? There have been a number of blog-related PRs that do not contain the front matter. Or, provide a front matter sample for each type of content page?

- Could the blog content use a separate style guide?

I think Dos and Dont's, as a table, work well when providing real examples -- and there are some mixed in with the new tables. Possibly pull out a few examples and put into a table so that the table looks like the other tables on the page. Do you think a basic list would work for the rest of the content? There is a sample of Dos and Donts from this [accessibility guide](https://developers.google.com/style/accessibility).

Reply:
@kbhawkey

- I plan to move the fort matter attributes info to https://kubernetes.io/docs/contribute/style/page-templates/

- Do's and Don't list work well. However, to keep it consistent with the rest of the Do and Don't in the table I have rephrased the guidelines.

- I would suggest that we try and keep all the guidelines in one page https://deploy-preview-14292--kubernetes-io-master-staging.netlify.com/docs/contribute/style/style-guide/ only rather than a separate guideline for blog content. We can always integrate the blog content guidelines with our existing guidelines.